### ISSUE TITLE
brave://apps has an ugly gray color as the background

### ISSUE THREAD

**Issue by urbenlegend** [brave/brave-browser#25736]
## Description 

Ever since https://github.com/brave/brave-browser/issues/16078 was fixed, `brave://apps` has been displaying an ugly gray color as the background that doesn't suit the application theme at all.

## Steps to Reproduce

   1. Navigate to `brave://apps`
   2. Notice that the background color is an ugly gray.

## Actual result:

![image](https://user-images.githubusercontent.com/241316/193312693-0e7a7950-b22c-42fc-93b0-c02b26a27c28.png)

## Expected result:
`brave://apps` background should match the selected Brave theme in settings.

## Reproduces how often: 

Every time

## Brave version (brave://version info)

1.44.101 Chromium: 106.0.5249.65 (Official Build) (64-bit) 

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Not tested
- Can you reproduce this issue with the nightly channel? Not tested

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by sungod12** [comment]
Hey, can I  start working on this issue ?

**Comment by bbondy** [comment]
Yes but I think the best place to fix this issue is in Chromium upstream. I don't think it is Brave specific.

**Comment by bbondy** [comment]
oh ok, so it sounds like it's brave specific then 👍

**Comment by rebron** [comment]
cc: @petemill

**Comment by urbenlegend** [comment]
Recently, Brave has been showing a completely black background instead of a gray one:
![image](https://github.com/brave/brave-browser/assets/241316/d008ec15-dc82-4cd6-97ff-8c2facda3b2a)

The background should follow the color theme instead of being dark all the time.

**Comment by ParagGhatage** [comment]
@urbenlegend   
I would like to work on this issue.
Will you assign this to me?

**Comment by urbenlegend** [comment]
@ParagGhatage I do not have permissions to do so. I am not a Brave employee btw.

**Comment by ParagGhatage** [comment]
> @ParagGhatage I do not have permissions to do so. I am not a Brave employee btw.

Thanks for the clarification.

**Comment by Omvinchhi** [comment]
is this still open ??

**Comment by Rishabh6395** [comment]
> Recently, Brave has been showing a completely black background instead of a gray one: ![image](https://private-user-images.githubusercontent.com/241316/323029825-d008ec15-dc82-4cd6-97ff-8c2facda3b2a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU1NTc2OTgsIm5iZiI6MTcyNTU1NzM5OCwicGF0aCI6Ii8yNDEzMTYvMzIzMDI5ODI1LWQwMDhlYzE1LWRjODItNGNkNi05N2ZmLThjMmZhY2RhM2IyYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkwNVQxNzI5NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mODdlZWZlMmI2YjIzZDYzMmJmYWM4NWIwMGI0YzNmMWIxOTBjMGM0ZjFlYmY2NmY1OTc4N2RhNzI4YmU4ZjEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-qkQrdDAXVYv-boG-RjKDMpzmuEEZa1dPDRzoExpw7A)
> 
> The background should follow the color theme instead of being dark all the time.



How do we reproduce this issue?

**Comment by urbenlegend** [comment]
@Rishabh6395 Simply navigate to `brave://apps` and you'll see the dark background.

**Comment by bsclifton** [comment]
@urbenlegend are you seeing black background or gray?

**Comment by urbenlegend** [comment]
I get black on both as well.

**Comment by mohanamisra** [comment]
@urbenlegend What would you like the page to look like?

This is what a typical new tab looks like for me:
![Image](https://github.com/user-attachments/assets/611e10f8-8b24-404c-83c5-915bcd1445c2)


Would you like the purple-pink-ish gradient to appear in the apps tab as well?

**Comment by urbenlegend** [comment]
I would like it if it just used the background color that is currently used in the background of Settings. If the "light" color is set under `brave://settings/appearance`, then it uses a light background. If "dark" is set, then it uses a dark background. If "Same as Linux" (or any other OS) is set, then it just follows the system's dark mode setting. It would be good if `brave://apps` used the exact same color and behavior for the sake of UI consistency.

![Image](https://github.com/user-attachments/assets/d7b16ad1-cec7-4f23-8d8c-4cb15924aef2)
![Image](https://github.com/user-attachments/assets/38bdac8c-1253-4750-8a77-a244e932487d)

**Comment by lbtbs** [comment]
No one assigned... :(

**Comment by aguscruiz** [comment]
Adding designs and implementation details here:
https://www.figma.com/design/2H6A1LEMPB2mUiAkCgnZCX/brave---apps?node-id=1-11646&t=iUWG95vCSoj60945-1

<img width="2735" height="2051" alt="Image" src="https://github.com/user-attachments/assets/8db11846-bba0-4f41-8cc2-4647b79e712c" />

**Comment by softsideof** [comment]
I can take this issue and will open a PR shortly.

**Comment by softsideof** [comment]
Implemented and opened PR: https://github.com/brave/brave-core/pull/35762
